torch/autograd/functional.py

```diff
@@ -433,6 +503,10 @@ def jacobian(func, inputs, create_graph=False, strict=False, vectorize=False):
             because this feature is still experimental, there may be performance
             cliffs. See :func:`torch.autograd.grad`'s ``batched_grad`` parameter for
             more information.
+        strategy (str, optional): Set to ``"forward-mode"`` or ``"reverse-mode"`` to
```

Comment:
I think I am happy with both strategy or mode.

One question though is if forward/reverse the right toggle here. Isn't a toggle that talks about the number of input/outputs of the function easier to understand by the end user?